doc/CHANGELOG.md

```diff
@@ -207,67 +226,73 @@ All notable changes to this project will be documented in this file.
   - Three new zinit annexes (i.e.
     [extensions](http://z-shell.org/zinit/wiki/Annexes/)):
 
-      - [z-a-man](https://github.com/zinit/z-a-man)
+    - [z-a-man](https://github.com/zinit/z-a-man)
 
-        Generates man pages and code-documentation man pages from plugin's README.md
-        and source files (the code documentation is obtained from
-        [Zshelldoc](https://github.com/z-shell/zshelldoc)).
+      Generates man pages and code-documentation man pages from plugin's README.md
+      and source files (the code documentation is obtained from
+      [Zshelldoc](https://github.com/z-shell/zshelldoc)).
 
-      - [z-a-test](https://github.com/zinit/z-a-test)
+    - [z-a-test](https://github.com/zinit/z-a-test)
 
-        Runs tests (if detected `test' target in a `Makefile` or any `*.zunit` files)
-        on plugin installation and non-empty update.
+      Runs tests (if detected `test' target in a `Makefile`or any`\*.zunit` files)
```

Comment:
Codacy has a fix for the issue: [Spaces inside code span elements](https://app.codacy.com/gh/z-shell/zinit/pullRequest?prid=8393798)
```suggestion
      Runs tests (if detected `test' target in a`Makefile`or any`\*.zunit` files)
```